index.html

```diff
@@ -160,6 +162,11 @@ <h4><b>{{ meetup.Day }}</b>/{{ meetup.Month }}/{{ meetup.Year }}<br><b>{{ meetup
           </a>
         </div>
       </div>
+      <div class="text-center" ng-show="totalPastMeetups() > loadedMeetups && $last!=true && $index==loadedMeetups">
+        <button type="button" class="btn btn-default btn-sm" ng-click="loadMoreMeetups()">
+          <span class="glyphicon glyphicon-refresh"></span> More Events
```

Comment:
For some reason ```glyphicon glyphicon-refresh``` on this span doesn't get displayed as it should.
Maybe it's a font or a css issue, can you check please?